fraim/workflows/code/triage_sarif_overlay.py

```diff
@@ -17,3 +19,12 @@ class ResultProperties(BaseSchema):
     attack_complexity: str = Field(description="Complexity required to exploit the vulnerability (Low/Medium/High)")
     attack_vectors: list[str] = Field(description="List of potential attack vectors for exploiting the vulnerability")
     remediation: str = Field(description="Recommended steps to remediate the vulnerability")
+    risk_type: str = Field(
```

Comment:
Copy/paste from risk flagger? The code SAST workflow doesn't have "risks", per se.